cassandra/cqlengine/named.py

```diff
@@ -86,6 +87,13 @@ class NamedTable(object):
 
     _partition_key_index = None
 
+    __connection__ = None
+    _connection = None
+
+    using = UsingDescriptor()
+
+    _get_connection = BaseModel._get_connection
```

Comment:
alright. I wasn't sure if the hybrid_method could be binded to the other class or something.